pkg/venafi/tpp/tpp.go

```diff
@@ -317,6 +317,8 @@ const (
 	urlResourceCleanPolicy            urlResource = "vedsdk/config/clearpolicyattribute"
 	urlResourceSshCertReq             urlResource = "vedsdk/SSHCertificates/request"
 	urlResourceSshCertRet             urlResource = "vedsdk/SSHCertificates/retrieve"
+	urlResourceSshCAPubKey            urlResource = "vedsdk/SSHCertificates/Template/Retrieve/PublicKeyData"
+	urlResourceSshCADetails           urlResource = "/vedsdk/SSHCertificates/Template/Retrieve"
```

Comment:
I think the first slash (/) is not needed.
```suggestion
	urlResourceSshCADetails           urlResource = "vedsdk/SSHCertificates/Template/Retrieve"
```

Reply:
fixed on the most recent commits